Gemfile

```diff
@@ -2,48 +2,48 @@ source 'https://rubygems.org'
 ruby '2.1.5'
 
 # the base rails libraries
-gem 'rails', '~> 3.2.16'
-gem 'thin'
 gem 'pg'
+gem 'rails', '~> 3.2.16'
 gem 'rails_12factor'
+gem 'thin'
 
 # for handling json objects with ruby
 gem 'json'
 
 # lets us use the ubiquitous jquery javascript library
```

Comment:
can we get rid of this line too? It's now misplaced but it doesn't really offer any info. It's pretty clear what `jquery-rails` does :)